Makefile

```diff
@@ -89,13 +89,14 @@ check:
 static: export GO111MODULE=on
 static: prepare tools
 	@ # Not running vet and fmt through metalinter becauase it ends up looking at vendor
-	tools/bin/goimports -w -d -format-only -local $(BR_PKG) $$($(PACKAGE_DIRECTORIES)) 2>&1 | $(CHECKER)
+	tools/bin/gofumports -w -d -format-only -local $(BR_PKG) $$($(PACKAGE_DIRECTORIES)) 2>&1 | $(CHECKER)
```

Comment:
Is `goimports` a subset of `gofumports`? I haven't found the relationship of them. Should we have them both?

Reply:
It's a stricter gofmt, also a drop-in replacement for goimports.

https://github.com/mvdan/gofumpt